src/VisualStudio/Core/Def/EditorConfigSettings/DataProvider/CodeStyle/CommonCodeStyleSettingsProvider.cs

```diff
@@ -109,6 +109,10 @@ private IEnumerable<CodeStyleSetting> GetNullCheckingCodeStyleOptions(AnalyzerCo
                 description: EditorFeaturesResources.Prefer_is_null_for_reference_equality_checks,
                 editorConfigOptions: options,
                 visualStudioOptions: visualStudioOptions, updater: updater, fileName: FileName);
+            yield return CodeStyleSetting.Create(option: CodeStyleOptions2.PreferParameterNullChecking,
```

Comment:
👍

Reply:
I think (since this is a C# only option) this should come from here: https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/CSharp/Impl/EditorConfigSettings/DataProvider/CodeStyle/CSharpCodeStyleSettingsProvider.cs

Reply:
I can't quite figure out the right set of config to use here. Have tried comparing with other csharp specific style options and am scratching my head. Could use some help with someone familiar with this area when possible.

Reply:
Seeing the following test failure with my current config

```log
[xUnit.net 00:00:31.11]     Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics.ConfigureSeverityLevel.IDEDiagnosticIDConfigurationTests.CSharp_VerifyIDECodeStyleOptionsAreConfigurable [FAIL]
  Failed Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics.ConfigureSeverityLevel.IDEDiagnosticIDConfigurationTests.CSharp_VerifyIDECodeStyleOptionsAreConfigurable [57 ms]
  Error Message:
   Missing entry:

# IDE0190
No editorconfig based code style option

Expected: False
Actual:   True
  Stack Trace:
     at Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics.ConfigureSeverityLevel.IDEDiagnosticIDConfigurationTests.<VerifyConfigureCodeStyleOptionsCore>g__ProcessDiagnosticIdAndOption|6_0(String diagnosticId, IOption optionOpt, String editorConfigString, <>c__DisplayClass6_0& ) in /_/src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs:line 689
   at Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics.ConfigureSeverityLevel.IDEDiagnosticIDConfigurationTests.VerifyConfigureCodeStyleOptionsCore(String expected, String languageName) in /_/src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs:line 644
   at Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics.ConfigureSeverityLevel.IDEDiagnosticIDConfigurationTests.CSharp_VerifyIDECodeStyleOptionsAreConfigurable() in /_/src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs:line 1047
```

I do not know what I should change here. It seems like `No editorconfig based code style option` is not what we should see in the editorconfig file.

Reply:
https://github.com/dotnet/roslyn/pull/58182#discussion_r780564873 is likely to be the reason :)